### ISSUE TITLE
version inconsistency to smaa

### ISSUE THREAD

**Issue by cguentherTUChemnitz** [scrawl/smaa-opengl#3]
The original smaa repository seems to have updated sources:
https://github.com/iryoku/smaa

They seem to have updated lookup textures as well as smaa shaders. For example a lot of macros are renamed. Currently there is direct GLSL support also in the original branch for the shaders, but no example implementation.

I would appreciate some clarification about the lookup textures. The https://github.com/iryoku/smaa project has tga files, as well as python scripts producing those tga files and printing the values direct to stdout. This https://github.com/scrawl OpenGL port uses raw textures instead. So i have no idea how to correctly use current lookup textures. My question would be how to use the printed output of the scripts using OpenGL? Do they have to be flipped first or are they plain usable for OpenGL textures. Does the color format also fit with this example port? How the raw textures are generated for this project?

**Comment by scrawl** [comment]
Unfortunately I don't remember the answer to any of these questions. All I can recall is that some issue with the textures was the main roadblock to getting it working. Maybe @Yours3lf would know more.

**Comment by cguentherTUChemnitz** [comment]
I got reasonable results after flipping the texture v-coordinates. But i have the problem that the results are not very consistent to the DX10 demo. As described here:
https://github.com/iryoku/smaa/issues/8

**Comment by sasmaster** [comment]
I ported this repo to my needs. Stripped all the deprecated OpenGL stuff. Yes , you should take care of coordinates flipping as you render into offscreen buffers. Make sure your screen quad/tri has correct UV coordinates. I actually do see plausible results.The only thing I wonder about is why albedo texture must have GL_FLOAT data type. The internal format is GL_RGBA8 so I can't see how GL_FLOAT helps here.

**Comment by sasmaster** [comment]
Indeed,I just tested with GL_UNSIGNED_BYTE for albedo and blend textures - I see no difference.
